google/cloud/aiplatform/jobs.py

```diff
@@ -36,20 +38,27 @@
     completion_stats as gca_completion_stats,
     custom_job as gca_custom_job_compat,
     explanation as gca_explanation_compat,
+    encryption_spec as gca_encryption_spec_compat,
     io as gca_io_compat,
     job_state as gca_job_state,
     hyperparameter_tuning_job as gca_hyperparameter_tuning_job_compat,
     machine_resources as gca_machine_resources_compat,
     manual_batch_tuning_parameters as gca_manual_batch_tuning_parameters_compat,
     study as gca_study_compat,
 )
+
+from google.cloud.aiplatform_v1.types import (
```

Comment:
I tried that at first and I got an import error saying `AttributeError: 'tuple' object has no attribute 'ModelDeploymentMonitoringObjectiveConfig'`. Not really sure what I did wrong because it seems like the types in `compat` are just aliased types from GAPIC, and I added all the necessary imports in the `__init__.py` files in `compat`.

Reply:
That's because they are being assigned to tuples in ` google/cloud/aiplatform/compat/__init__.py`. Unenvelope them and that exception should go away. 

Preference to go through compat so the API version will get switched when  DEFAULT_VERSION is runtime patched.

Reply:
fixed import statements